### ISSUE TITLE
Server throws "Error writing to socket" on disconnection.

### ISSUE THREAD

**Issue by EstebanAtHisComputer** [5Mixer/mphx#40]
First of all, congratulations for the awesome work!
I'm trying to use the `onConnectionClose` callback to handle players leaving the game, and, while it works, it keeps throwing an exception: `NetSock.hx:41: Error writing to socket: Custom(std@socket_send)`.
As I say, the callback works, but having the server full of exceptions looks bad.
This is the code I'm using:

```
s.onConnectionClose = function(s1:String, sender:IConnection) {
    _playersNumber--;

    var player = players.get(sender);
    trace("Player " + player.id + " disconnected");

    s.broadcast("Disconnect", {
        id:player.id
    });

    players.remove(sender);
}
```

**Comment by 5Mixer** [comment]
Hi there, thanks for raising the issue. Sorry it took some time to get it this. I just ran the `simple > basic` example, and it seemed to work.
![working disconnections](https://cloud.githubusercontent.com/assets/8501694/17691920/951b8512-63da-11e6-8ceb-d62d8edf7301.gif)
So I was a little confused, however after looking at your code, I noticed that you broadcast that a player had left. Realizing that this was trying to send a message to a now non-existant client, it was just a case of switching around the order of lines. 75317cabab0

Tell me if the problem happens to continue, otherwise this should be resolved :smile:
